charts/incubator/external-dns/templates/_args.tpl

```diff
@@ -0,0 +1,48 @@
+{{- define "externaldns.args" -}}
+args:
+  {{- with .Values.externaldns.provider }}
+  - --provider={{ . }}
+  {{- end -}}
+  {{- with .Values.externaldns.zoneidfilter }}
+  - --zone-id-filter={{ . }}
+  {{- end -}}
+  {{- with .Values.externaldns.cloudflareproxied }}
+  - --cloudflare-proxied={{ tpl . $ }}
+  {{- end }}
+  - --log-level={{ .Values.externaldns.logLevel }}
+  - --log-format={{ .Values.externaldns.logFormat }}
+  - --interval={{ .Values.externaldns.interval }}
+  {{- if .Values.externaldns.triggerLoopOnEvent }}
+  - --events
+  {{- end -}}
+  {{- range .Values.externaldns.sources }}
+  - --source={{ . }}
+  {{- end -}}
+  {{- with .Values.externaldns.policy }}
+  - --policy={{ . }}
+  {{- end -}}
+  {{- with .Values.externaldns.registry }}
+  - --registry={{ . }}
+  {{- end -}}
+  {{- if .Values.externaldns.txtOwnerId }}
+  - --txt-owner-id={{ .Values.externaldns.txtOwnerId }}
+  {{- end -}}
+  {{- if .Values.externaldns.txtPrefix }}
+  - --txt-prefix={{ .Values.externaldns.txtPrefix }}
+  {{- end -}}
+  {{- if and (eq .Values.externaldns.txtPrefix "") (ne .Values.externaldns.txtSuffix "") }}
+  - --txt-suffix={{ .Values.externaldns.txtSuffix }}
+  {{- end -}}
+  {{- if .Values.externaldns.namespaced }}
+  - --namespace={{ .Release.externaldns.Namespace }}
```

Comment:
I think we should use the template we have for namespace here (https://github.com/truecharts/library-charts/blob/5970e7e9cda0acd2bfa7bd6ab0576e49f1f85de0/library/common/templates/lib/metadata/_namespace.tpl).

Reply:
I'm lost there lol, like wouldn't it just pull the normal NS like other charts